spec/dummy/config/application.rb

```diff
@@ -7,7 +7,7 @@
 require "active_model/railtie"
 require "active_job/railtie"
 # require "active_record/railtie"
-require "active_storage/engine"
+# require "active_storage/engine"
```

Comment:
Cant his just be deleted?

Reply:
Yes.  We commented out the other options, so I did the same with this one.